wp/sections/diKTat.tex

```diff
@@ -0,0 +1,5 @@
+\subsection{What is diKTat?}
+DiKTat - is a formal strict code style (\url{https://github.com/cqfn/diKTat}) and a linter with a set of rules that implement this code style for Kotlin language. Basically, it is a collection of Kotlin code style rules implemented as AST visitors on top of KTlint framework (\url{https://github.com/pinterest/ktlint}). Diktat warns and fixes code style errors and code smells based on configuration file. DiKTat is a highly configurable framework, that can be extended further by adding custom rules. It can be run as command line application or with maven or gradle plugins. In this paper, we will explain how DiKTat works, describes advantages and disadvantages and how it differs from other static analyzers.
+
+\subsection{Why diKTat?}
```

Comment:
We have understood that we simply will need to implement lot of configurable rules that are missing in other static analyzers. But we decided not to create "yet another one framework with dirty hacks" and decided to implement our set of rules on the top of a Ktlint framework that we are using as a dependency in Diktat